Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,54 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
+This is the default end-of-checkout script.
+
+It will leave the telescope in position to start taking afternoon calibraitons. This is az = 205.7 elev = 30
+If you are not sure how you want to leave the telescope, consider running this script as it will perform a few more useful checks of the system.
+
+
+Procedure
+==========
+
+Be sure you have run previously the following scripts:
```

Comment:
For instance, this part would go under pre-requisites: The Daytime Checkout must have be run successfully.